docs/getting-started.md

```diff
@@ -44,3 +44,13 @@ You can install an editor extension to display Glint's diagnostics inline in you
 - Install the [VS Code Extension](https://marketplace.visualstudio.com/items?itemName=typed-ember.glint-vscode).
 
 ![A type error being shown inline for a template file in VS Code](https://user-images.githubusercontent.com/108688/111076679-995c2300-84ed-11eb-934a-3a29f21be89a.png)
+
+To get Ember/Glimmer and TypeScript working together, Glint creates a separate TS language service instance patched with Ember-specific support. To prevent invalid or duplicate diagnostics you need to disable VSCode's built-in TS language service in your project's workspace only by following these steps:
+
+1. In your project workspace, bring up the command palette with `Ctrl + Shift + P` (macOS: `Cmd + Shift + P`).
+2. Type `built` and select "Extensions: Show Built-in Extensions".
+3. Type `typescript` in the extension search box (do not remove `@builtin` prefix).
```

Comment:
You can get there a bit more directly:
```suggestion
1. In your project workspace, bring up the extensions sidebar `Ctrl + Shift + X` (macOS: `Cmd + Shift + X`).
2. Type `@builtin typescript` in the extension search box
```

Reply:
Good point, that saves a step. To be clear, I was inspired by the [Volar docs](https://vuejs.org/guide/typescript/overview.html#volar-takeover-mode) on these. (quite literally 😬)

Adjusted and updated in both places.